requirements.yaml

```diff
@@ -14,3 +14,6 @@ dependencies:
     version: 0.2.2
     repository: https://kubernetes-charts-incubator.storage.googleapis.com/
     condition: docker-registry.enabled
+  - name: external-dns
```

Comment:
Because there could be `NodePort` type, this dependency needs a condition to download it only if service type is LoadBalancer

Reply:
Yes. Thanks.